src/features/ballot/components/BallotConfirmation.tsx

```diff
@@ -1,88 +1,136 @@
+        </p>
+        <div>
+          {allocations.map((project) => {
+            return (
+              <div className="border-b border-gray-200 py-3">
+                <ProjectAvatarWithName
+                  id={project.projectId}
+                  allocation={project.amount}
+                />
+              </div>
+            );
+          })}
+        </div>
+        <div className="mt-4 flex w-full justify-end">
+          <h4>Total votes allocated:</h4>
+          <p>{sum}</p>
+        </div>
+      </div>
+      <Notification
+        variant="block"
+        title="Results will be available after tallying."
+        content={format(config.resultsAt, "d MMM yyyy hh:mm")}
+      />
+      {appState === EAppState.VOTING && (
         <Card>
-          <div className="mb-6">
-            <h5 className="mb-3 text-2xl font-bold">
-              Here&apos;s how you voted!
-            </h5>
-            <div className="flex items-center gap-2 text-gray-600 dark:text-gray-300">
-              <Lock className="h-4 w-4 " />
-              <p>Your vote will always be private</p>
-            </div>
-          </div>
-          <div className="flex items-center justify-between border-b py-3 text-gray-600 dark:border-gray-700 dark:text-gray-300">
-            <p>Project name</p>
-            <p>{config.tokenName} allocated by you</p>
+          <div className="flex-3 flex flex-col gap-4">
+            <b className="font-mono text-2xl uppercase">
+              Wanna change your mind?
+            </b>
+            <p className="text-gray-400">
+              Your can edit your ballot and resubmit it anytime during the
+              voting period.
+            </p>
           </div>
-
-          <section className="max-h-[480px] overflow-y-scroll">
-            {votes && <AllocationList votes={votes} />}
-          </section>
-        </Card>
-        <Card>
           <div>
-            <h5 className="mb-3 text-2xl font-bold">
-              Help us improve next round of RetroPGF
-            </h5>
-            <p className="mb-6 text-gray-700 dark:text-gray-300">
-              Your anonymized feedback will be influential to help us iterate on
-              Optimism&apos;s RetroPGF process.
-            </p>
-            <Button
-              variant="primary"
-              as={Link}
-              target="_blank"
-              href={feedbackUrl}
-            >
-              Share your feedback
+            <Button variant="primary" as={Link} href="/ballot">
+              Edit my ballot
             </Button>
           </div>
         </Card>
-      </div>
+      )}
+      <Card>
+        <div className="flex-3 flex flex-col gap-4">
+          <b className="font-mono text-2xl uppercase">
+            Help us improve our next round of {config.eventName}
+          </b>
+          <p className="text-gray-400">
+            Your anonymized feedback will be influential to help us iterate on
+            Optimism’s RetroPGF process.
```

Comment:
should we remove Optimism mention here?

Reply:
also, are we actually saving any feedback?

Reply:
currently we just link the feedback to create a github issue, maybe should create a google form for it?

Reply:
for updating feedback link, I opened another issue for it: https://github.com/privacy-scaling-explorations/maci-rpgf/issues/166